app/src/main/java/org/greenstand/android/TreeTracker/capture/TreeCaptureScreen.kt

```diff
@@ -33,7 +27,8 @@ import org.greenstand.android.TreeTracker.view.*
 fun TreeCaptureScreen(
     profilePicUrl: String,
 ) {
-    val viewModel: TreeCaptureViewModel = viewModel(factory = TreeCaptureViewModelFactory(profilePicUrl))
+    val viewModel: TreeCaptureViewModel =
+        viewModel(factory = TreeCaptureViewModelFactory(profilePicUrl))
```

Comment:
Please look over your PRs after you make them, to make sure there are no formatting issues like this. It makes it harder to review code when there are many little formatting changes that distract from what you actually did.